docs/PLUGIN_CATALOG.md

```diff
@@ -1,85 +1,87 @@
+| Nova | Collector | Intel | Collects OpenStack Nova module metrics | [snap-plugin-collector-nova](https://github.com/intelsdi-x/snap-plugin-collector-nova) | [GitHub](https://github.com/intelsdi-x/snap-plugin-collector-nova/releases/latest) |
+| OSv | Collector | Intel | Collects osv performance metrics | [snap-plugin-collector-osv](https://github.com/intelsdi-x/snap-plugin-collector-osv) | [GitHub](https://github.com/intelsdi-x/snap-plugin-collector-osv/releases/latest) |
+| Openfoam | Collector | Intel | Collects OpenFOAM performance metrics | [snap-plugin-collector-openfoam](https://github.com/intelsdi-x/snap-plugin-collector-openfoam) | Not yet released |
+| PCM | Collector | Intel | Collects Intel Performance Counter Metrics (PCM) | [snap-plugin-collector-pcm](https://github.com/intelsdi-x/snap-plugin-collector-pcm) | [GitHub](https://github.com/intelsdi-x/snap-plugin-collector-pcm/releases/latest) |
+| PSUtil | Collector | Intel | Collects Linux psutil metrics | [snap-plugin-collector-psutil](https://github.com/intelsdi-x/snap-plugin-collector-psutil) | [GitHub](https://github.com/intelsdi-x/snap-plugin-collector-psutil/releases/latest) |
+| Perfevents | Collector | Intel | Collects Linux cgroups perf metrics | [snap-plugin-collector-perfevents](https://github.com/intelsdi-x/snap-plugin-collector-perfevents) | [GitHub](https://github.com/intelsdi-x/snap-plugin-collector-perfevents/releases/latest) |
+| Ping | Collector | raintank |  | [snap-plugin-collector-ping](https://github.com/raintank/snap-plugin-collector-ping) | [GitHub](https://github.com/raintank/snap-plugin-collector-ping/releases/latest) |
+| Processes | Collector | Intel | Collects Linux process metrics from /proc/processes | [snap-plugin-collector-processes](https://github.com/intelsdi-x/snap-plugin-collector-processes) | [GitHub](https://github.com/intelsdi-x/snap-plugin-collector-processes/releases/latest) |
+| Procnum | Collector | raintank | No description. | [snap-plugin-collector-procnum](https://github.com/raintank/snap-plugin-collector-procnum) | [GitHub](https://github.com/raintank/snap-plugin-collector-procnum/releases/latest) |
+| Procstat | Collector | Staples-Inc | Collect process metrics using the snap telemetry framework | [snap-plugin-collector-procstat](https://github.com/Staples-Inc/snap-plugin-collector-procstat) | [GitHub](https://github.com/Staples-Inc/snap-plugin-collector-procstat/releases/latest) |
+| RabbitMQ | Collector | Intel | Collects RabbitMQ metrics from management API | [snap-plugin-collector-rabbitmq](https://github.com/intelsdi-x/snap-plugin-collector-rabbitmq) | [GitHub](https://github.com/intelsdi-x/snap-plugin-collector-rabbitmq/releases/latest) |
+| Redfish | Collector | Intel | Collects metrics from Redfish API  | In development by [@candysmurf](https://github.com/candysmurf) | No release date - [please open an issue](https://github.com/intelsdi-x/snap/issues) if you'd like to see this |
```

Comment:
Manually - what's your thought on it? The reason I kept it is that it was listed previously.